nautobot/extras/utils.py

```diff
@@ -80,6 +82,40 @@ def get_choices(self):
         return [(f"{ct.app_label}.{ct.model}", ct.pk) for ct in ContentType.objects.filter(self.get_query())]
 
 
+@deconstructible
+class ModelSubclassesQuery:
```

Comment:
Thinking about this (yet again, sorry), this isn't perhaps the most Pythonic way to approach the problem. Should we think about instead using a duck-typing approach here (something like `if hasattr(_class, "tags") and isinstance(_class.tags, TaggableManager)`) instead of the `if issubclass(_class, klass)` check? Maybe a plugin might have a use case for enabling tags on a model that **isn't** a PrimaryModel because it doesn't support change-logs and dynamic-groups for whatever reason...